lib/config/wrapper.go

```diff
@@ -55,11 +55,16 @@ var errTooManyModifications = errors.New("too many concurrent config modificatio
 // configuration (e.g. calling Wrapper.SetFolder), that are also acquired in any
 // methods of the Committer interface.
 type Committer interface {
-	VerifyConfiguration(from, to Configuration) error
 	CommitConfiguration(from, to Configuration) (handled bool)
 	String() string
 }
 
+// A Verifier can determine if a new configuration is acceptable.
+// See the description for Committer, above.
+type Verifier interface {
+	VerifyConfiguration(from, to Configuration) error
+}
```

Comment:
This is probably missing String() which would be used printing the debug message.

Also, the fact that we do this magically concerns me, because its enough to change the argument type and things will stop working, so for this that do implement it, we should have a compile time assertion that they satisfy the interface.

Reply:
Or alternative, provide a NoopVerifier that all thing embed (which I guess just brings us to existing behaviour)

Reply:
Type-asserting to Verifier doesn't remove the String method. Debugln passes its argument to Sprintln, which type-asserts to fmt.Stringer.

I put in the assertions, though.

Reply:
I don't think the concern is about the String() method being removed or inaccessible, but that the code that runs the Verifier expects it to always be there so it could / should be part of the interface. However it's just used for debugging, so 🤷